docs/integrations/aws_lambda.md

```diff
@@ -45,23 +45,24 @@ pip install \
 ### `no module named 'pydantic_core._pydantic_core'`
 
 The
-```
+
+```output
 no module named `pydantic_core._pydantic_core`
 ```
 
 error is a common issue that indicates you have installed `pydantic` incorrectly. To debug this issue, you can try the following steps (before the failing import):
 
 1. Check the contents of the installed `pydantic-core` package. Are the compiled library and its type stubs both present?
 
-```python {test="skip" lint="skip"}
-from importlib.metadata import files
-print([file for file in files('pydantic-core') if file.name.startswith('_pydantic_core')])
-"""
-[PackagePath('pydantic_core/_pydantic_core.pyi'), PackagePath('pydantic_core/_pydantic_core.cpython-312-x86_64-linux-gnu.so')]
-"""
-```
+    ```python {test="skip" lint="skip"}
+    from importlib.metadata import files
+    print([file for file in files('pydantic-core') if file.name.startswith('_pydantic_core')])
+    """
+    [PackagePath('pydantic_core/_pydantic_core.pyi'), PackagePath('pydantic_core/_pydantic_core.cpython-312-x86_64-linux-gnu.so')]
+    """
+    ```
 
-You should expect to see two files like those printed above. The compile library file will be a .so or .pyd with a name that varies according to the OS and Python version.
+    You should expect to see two files like those printed above. The compile library file will be a .so or .pyd with a name that varies according to the OS and Python version.
 
 2. Check that your lambda's Python version is compatible with the compiled library version found above.
```

Comment:
This list element also has a code block that can be indented with two spaces to match the others.